go.mod

```diff
@@ -139,8 +139,6 @@ replace google.golang.org/grpc => google.golang.org/grpc v1.33.2
 
 replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 
-replace github.com/tecbot/gorocksdb => github.com/cosmos/gorocksdb v1.2.0
```

Comment:
go mod was failing from this line.. dont think we actually need it tho?

Reply:
Hmmm.... https://github.com/regen-network/regen-ledger/pull/677

Reply:
after looking at it again, i realized we already have cosmos/gorocksdb under the `require` so no need to replace it anymore